development/VehicleConfigParams.yaml

```diff
@@ -100,8 +100,8 @@ vehicle_participant_type: "vehicle:car"
 config_speed_limit: 0.0
 
 # Parameters to enable either ROS 1 or ROS 2 (or both) rosbag logging
-use_ros1_rosbag: true
-use_ros2_rosbag: false
+use_ros1_rosbag: false
+use_ros2_rosbag: true
```

Comment:
nit: Lets turn rosbag recording off by default

Reply:
I think we should leave it on because this is development folder and we usually need the data debug